moonbeam-types-bundle/package-lock.json

```diff
@@ -5,321 +5,65 @@
   "requires": true,
   "dependencies": {
     "@babel/runtime": {
-      "version": "7.16.3",
-      "resolved": "https://registry.npmjs.org/@babel/runtime/-/runtime-7.16.3.tgz",
-      "integrity": "sha512-WBwekcqacdY2e9AF/Q7WLFUWmdJGJTkbjqTjoMDgXkVZ3ZRUvOPsLb5KdwISoQVsbP+DQzVZW4Zhci0DvpbNTQ==",
+      "version": "7.16.7",
```

Comment:
Please revert those changes if not intended

Reply:
The package-lock changes were needed. The reason is that I needed version 6.12.1 to be able to decode and insert a message as if it was coming from statemine (in typescript). I will revisit, but I struggled with this file to be able to do that